src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1748,11 +1736,13 @@ private BetterResult BetterConversionFromExpression(BoundExpression node, TypeSy
         {
             okToDowngradeToNeither = false;
 
-            if (considerRefKinds)
+            var argumentKind = node.Kind;
+            if (considerRefKinds || argumentKind == BoundKind.OutVariablePendingInference || argumentKind == BoundKind.OutDeconstructVarPendingInference)
```

Comment:
The behavior we want arises from the specification for exactly matching expression. I will implement it that way, now that we have that tentative spec language.

---
In reply to: [80791006](https://github.com/dotnet/roslyn/pull/13999#discussion_r80791006) [](ancestors = 80791006)